src/main.rs

```diff
@@ -264,6 +268,9 @@ fn main() {
                     args.concurrency,
                     args.allow_missing_relations_in_first_blocks,
                 ));
+
+                let view_client = indexer.client_actors().0;
```

Comment:
I propose we add comments before `listen_blocks` spawn that we're spawning regular indexer job and before this spawn that we're running cirulating supply computation (just to help us to distinguish stuff we're running)